hydrocarbon_problem/api/aspen_api.py

```diff
@@ -0,0 +1,146 @@
+from typing import Tuple
+import retry.api as retry
+import numpy as np
+# from sqlalchemy import column
+from hydrocarbon_problem.api.Simulation import Simulation
+
+from hydrocarbon_problem.api.api_base import BaseAspenDistillationAPI
+from hydrocarbon_problem.api.types_ import StreamSpecification, ColumnInputSpecification, \
+    ColumnOutputSpecification, ProductSpecification, PerCompoundProperty
+
+PATH = 'C:/Users/s2399016/Documents/Aspen-RL_v2/Aspen-RL/hydrocarbon_problem/AspenSimulation/HydrocarbonMixture.bkp'
+
+
+class AspenAPI(BaseAspenDistillationAPI):
+    def __init__(self):
+        self._flowsheet: Simulation = Simulation(PATH=PATH, VISIBILITY=True)
+        self._feed_name: str = "S1"
+        self._tops_name: str = "S2"
+        self._bottoms_name: str = "S3"
+        self._name_to_aspen_name = PerCompoundProperty(ethane="ETHANE",
+                                                       propane="PROPANE",
+                                                       isobutane="I-BUTANE",
+                                                       n_butane="N-BUTANE",
+                                                       isopentane="I-PENTAN",
+                                                       n_pentane="N-PENTAN")
+        
+    def set_input_stream_specification(self, stream_specification: StreamSpecification) -> None:
+        """Sets the input stream to a column to fit the stream specification"""
+        # Defining the Thermodynamic Properties
+        self._flowsheet.STRM_Temperature(self._feed_name,  stream_specification.temperature)
+        self._flowsheet.STRM_Pressure(self._feed_name, stream_specification.pressure)
+        # Defining the Stream Composition for the Feed
+        self._flowsheet.STRM_Flowrate(self._feed_name, self._name_to_aspen_name.ethane,     stream_specification.molar_flows.ethane)
```

Comment:
Is it possible to set all of these flowrates at once instead of making the repeated call to aspen?